lib/discovery-store-model.js

```diff
@@ -296,7 +296,9 @@ const buildDiscoveryStoreBibs = async (bibs) => {
         .then(attachRecapCustomerCodes)
     })
   )
-  await parseDatesAndCache(bibsPlusItemsAndRecapCodes)
+  if (!process.env.DISABLE_TIMETWISTER) {
```

Comment:
I think this may need to be `if (process.env.DISABLE_TIMETWISTER !== 'true')` because it's an env var and will be a string